front-dex/src/app/components/card/card.component.ts

```diff
@@ -8,16 +8,20 @@ import Pokemon from 'src/app/models/interfaces/Pokemon';
 })
 export class CardComponent implements OnInit {
   @Input() pokemon: Pokemon;
-  pokemonStorage: string = 'pokemon';
+  @Output() picked: EventEmitter<any> = new EventEmitter();
```

Comment:
Can you change the type to a specific type?

Reply:
nee.